.github/workflows/node-test.yml

```diff
@@ -0,0 +1,92 @@
+name: Test NodeJS
+
+on:
+  workflow_call:
+    inputs:
+      gh-node-version:
+        required: true
+        type: string
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}-${{ inputs.gh-node-version }}
+  cancel-in-progress: true
+
+jobs:
+  reset-badge:
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Reset ${{ inputs.gh-node-version }} badge"
+        uses: RubbaBoy/BYOB@v1.2.1
```

Comment:
nit: since this is a third-party action, we should replace tag with git commit in all places it is used